src/domains/chart/components/abstract-chart.tsx

```diff
@@ -45,6 +45,7 @@ interface Props {
   unitsCurrent: string
   viewAfter: number,
   viewBefore: number,
+  requestedViewRange: [number, number]
```

Comment:
Could we use named tuples types? For this case it is totaly ok to write this way, but sometimes it is hard to understand
why not 
```typescript
number[]
but
[number, number]
```
Also as far as it is range, type of range could be used over the project.

Reply:
good point! I'll change it to reuse a common type. Btw. i used it as tuple, just to have stricter type where it's possible